### PR TITLE
[DUOS-2340][risk=no] Switch to Material UI 5 everywhere

### PR DESCRIPTION
**Pull request by fboulnois**
### Addresses

https://broadworkbench.atlassian.net/browse/DUOS-2340

### Summary

Currently we rely on a combination of Material UI 4 and 5 in different places. Unify on using Material UI 5.

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/components/library_card_table/LibraryCardTable.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import lockIcon from '../../images/lock-icon.png';
 import LibraryCardFormModal from '../modals/LibraryCardFormModal';
 import { LibraryCard } from '../../libs/ajax';
 import ConfirmationModal from '../modals/ConfirmationModal';
-import {Delete, Update} from '@material-ui/icons';
+import {Delete, Update} from '@mui/icons-material';
```
Comment (rushtong) [inline src/components/library_card_table/LibraryCardTable.js:15]: There is a non-trivial change to the icon sizes here. New view is in the foreground, dev is in the background.

Screenshot 2023-02-14 at 8 32 57 AM

File: src/components/manage_dac_table/ManageDacTableCellData.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import {styles} from './ManageDacTable';
 import {a, label, h, div} from 'react-hyperscript-helpers';
 import TableIconButton from '../../components/TableIconButton';
 import {Styles} from '../../libs/theme';
-import {Delete, Edit} from '@material-ui/icons';
+import {Delete, Edit} from '@mui/icons-material';
```
Comment (rushtong) [inline src/components/manage_dac_table/ManageDacTableCellData.js:6]: Similar issue here:

Screenshot 2023-02-14 at 8 38 21 AM

**Review by rushtong — APPROVED** [review]
I found two screens where there were minor icon differences. I don't think those should hold up this PR though. Feel free to update those here or in a new ticket. 👍🏽

File: src/components/library_card_table/LibraryCardTable.js (hunk 1 above)
Comment (fboulnois) [reply src/components/library_card_table/LibraryCardTable.js:15]: Yes, you're right. I'll address this and add some tests around the mui icons today.

**Review by rushtong — APPROVED** [review]
Thanks for the updates! 👍🏽 
As a side note, I've held off approving https://github.com/DataBiosphere/duos-ui/pull/2052 until this went through.

**Review by nmalfroy — APPROVED** [review]
looks good!

**Pull request merged**